test/struct.js

```diff
@@ -379,4 +379,29 @@ describe('Struct', function () {
 
   })
 
+  describe('usePack()', function () {
+    var size = {
+        unpack: { ia32: 8, x64: 16 },
```

Comment:
hmm... basically `padding` depends on compiler. so i think, we should define struct in c file, and compare this.
`ref.sizeof.pointer` is only can check pack value.

this test case should change to
```javascript
{
  unpack: bindings.sizeof_with_padding_struct,
  pack: ref.sizeof.char + ref.sizeof.pointer // or bindings.sizeof_without_padding_struct
}
```
right?